src/components/OtherProjects.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable react/jsx-closing-tag-location */
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components'
+import { InnerWrapper, OuterWrapper, A, P } from 'StyledComponents/GlobalStyles';
+import { SectionHeading } from 'StyledComponents/HeadlineStyles';
+import { Tag, TagContainer } from 'StyledComponents/TagStyles.js'
+
+const OtherProjects = () => {
+  const [projects, SetProjects] = useState([])
+
+  useEffect(() => {
+    fetch('https://api.github.com/users/jesshansson/repos')
+      .then((res) => res.json())
+      .then((data) => SetProjects(data))
+  })
+
+  const filteredProjects = projects.filter((project) => (project.name !== 'project-business-site') && (project.name !== 'project-news-site') && (project.name !== 'project-react-portfolio') && (project.name !== 'project-happy-thoughts') && (project.name !== 'project-survey') && (project.name !== 'jessicahansson') && (project.name !== 'project-portfolio') && (project.name !== 'project-movies'))
```

Comment:
The line is a bit long and therefore very hard to read, consider splitting it up into more lines.